syntax = "proto3";

service Booking {
    rpc GetBooking(Empty) returns (stream BookingData) {}
    rpc GetBookingByUser(UserId) returns (BookingData) {}
}

message UserId {
    string id = 1;
}

message BookingData {
    string userid = 1;
    repeated DateData dates = 2;
}

message DateData {
    string date = 1;
    repeated string movieid = 2;
}

message Empty {
}

